daemon/algod/api/server/router_test.go

```diff
@@ -178,6 +191,6 @@ func (s *TestSuite) TestGetTransactionByID() {
 	assert.Equal(s.T(), callsBefore+1, s.calls)
 }
 func TestTestSuite(t *testing.T) {
-
+	partitiontest.PartitionTest(s)
```

Comment:
I think you would put partition here or in each of the tests, but not both. When we call `suite.Run` it will run all of the above tests.